crates/ruff/src/rules/flake8_use_pathlib/rules/path_constructor_current_directory.rs

```diff
@@ -0,0 +1,75 @@
+use ruff_diagnostics::{AlwaysAutofixableViolation, Diagnostic, Edit, Fix};
+use ruff_macros::{derive_message_formats, violation};
+use rustpython_parser::ast::{Constant, Expr, ExprCall, ExprConstant, Ranged};
+
+use crate::checkers::ast::Checker;
+use crate::registry::AsRule;
+
+/// ## What it does
+/// Detects pathlib `Path` objects that are initialized with the default current directory argument.
+///
+/// ## Why is this bad?
+/// The `Path()` constructor defaults to the current directory. There is no need to pass the current directory (`"."`) explicitly.
```

Comment:
nit: we wrap documentation at 80 characters
```suggestion
/// The `Path()` constructor defaults to the current directory. There is no
/// need to pass the current directory (`"."`) explicitly.
```